standard_robot_pp_ros2:
  ros__parameters:
    device_name: /dev/ttyACM0
    baud_rate: 115200
    flow_control: none
    parity: none
    stop_bits: "1"
    offset_timestamp: 0.008
